application/src/main/kotlin/net/corda/v5/application/crypto/CompositeKeyNodeAndWeight.kt

```diff
@@ -0,0 +1,10 @@
+package net.corda.v5.application.crypto
+
+import java.security.PublicKey
+
+data class CompositeKeyNodeAndWeight(val node: PublicKey, val weight: Int = 1) {
```

Comment:
As a DTO that class can be moved to corda-crypto

Reply:
now moved to corda-api